src/renderer/components/+custom-resources/crd-list.tsx

```diff
@@ -33,6 +33,15 @@ export class CrdList extends React.Component {
     return crdGroupsUrlParam.get();
   }
 
+  get items() {
+    const selectedGroups = this.groups;
+    const storeItems = crdStore.contextItems;
+
+    return selectedGroups.length ?
+      storeItems.filter(item => selectedGroups.includes(item.getGroup())) :
+      storeItems;
```

Comment:
I find having the `?` and `:` at the front of the line easier to read in passing.
```suggestion
    return selectedGroups.length
      ? storeItems.filter(item => selectedGroups.includes(item.getGroup()))
      : storeItems;
```